dm/worker/worker.go

```diff
@@ -763,6 +763,7 @@ func (w *Worker) copyConfigFromWorker(cfg *config.SubTaskConfig) {
 	cfg.ServerID = w.cfg.ServerID
 	cfg.RelayDir = w.cfg.RelayDir
 	cfg.EnableGTID = w.cfg.EnableGTID
+	cfg.SyncerConfig.EnableGTID = w.cfg.EnableGTID
```

Comment:
It seems this line is not necessary because both `cfg.EnableGTID` and `cfg.SyncerConfig.EnableGTID` are exactly the same variable.

```go
type SubTaskConfig struct {
    ...
    SyncerConfig   // Syncer configuration
    ...

this is not a problem in this PR.
}
```

Reply:
removed in 5605ab8